### PR TITLE
feact/Showing the appointment section updated

### PR DESCRIPTION
**Pull request by rohitja26**
Done the changes to the Appointment section as of now the user needs to login first to book an appointment.

### CODE REVIEW

File: Frontend/src/App.jsx [hunk 1]
```diff
@@ -1,64 +1,162 @@
...
-    { id: 7, name: "Dr. Wilson", specialty: "General practitioner", patients: 250, appointments: 620, experience: 12, qualifications: "MD, PHD", image: "https://randomuser.me/api/portraits/women/6.jpg", bio: "Dr. Wilson is an experienced general practitioner providing comprehensive primary care and preventive medicine for patients of all ages." },
-    { id: 8, name: "Dennis Schmidt", specialty: "Anesthesiologist", patients: 350, appointments: 750, experience: 17, qualifications: "MD, PHD", image: "https://randomuser.me/api/portraits/men/26.jpg", bio: "Dr. Dennis Schmidt is an experienced general practitioner providing comprehensive primary care and preventive medicine for patients of all ages." },
-    { id: 9, name: "Reginald Bennett", specialty: "Ophthalmologist", patients: 150, appointments: 450, experience: 15, qualifications: "MD, PHD", image: "https://randomuser.me/api/portraits/men/18.jpg", bio: "Dr. Reginald Bennett is an experienced general practitioner providing comprehensive primary care and preventive medicine for patients of all ages." },
+    {
+      id: 1,
```
Comment (yazdanhaider) [inline Frontend/src/App.jsx:26]: Please use Prettier for code formatting to ensure consistency and readability

File: Frontend/src/components/Appointments.jsx [hunk 2]
```diff
@@ -29,23 +38,37 @@ const Appointments = () => {
     ]);
 
     setAppointments([
-      { id: 1, patientName: "John Doe", doctorId: 1, appointmentDate: new Date(), notes: "Regular checkup", status: "Scheduled" },
-      { id: 2, patientName: "Jane Smith", doctorId: 2, appointmentDate: new Date(Date.now() + 86400000), notes: "Follow-up", status: "Scheduled" },
+      {
+        id: 1,
```
Comment (yazdanhaider) [inline Frontend/src/components/Appointments.jsx:42]: Please use Prettier for code formatting to ensure consistency and readability.

**Review by yazdanhaider — CHANGES_REQUESTED** [review]
Please use Prettier for code formatting to ensure consistency and readability.

File: Frontend/src/components/Appointments.jsx (hunk 2 above)
Comment (rohitja26) [reply Frontend/src/components/Appointments.jsx:42]: already used prettier for formatting and it is giving like the above only

Comment (yazdanhaider) [reply Frontend/src/components/Appointments.jsx:42]: Chek this out!!

Screenshot 2024-10-23 at 4 41 59 PM

File: Frontend/src/App.jsx (hunk 1 above)
Comment (rohitja26) [reply Frontend/src/App.jsx:26]: what is the problem I am not getting what I need to do as the code is formatted perfectly from my end and this formatted with the prettier

**Comment by rohitja26** [comment]
I have raised one more issue please see and assign me with that

**Pull request merged**